docs/modules/ROOT/pages/getting-started/configuration.adoc

```diff
@@ -0,0 +1,257 @@
+= Configuring KubeArchive
+
+This document explains how to configure KubeArchive so you can archive 
+resources and query them.
+
+== Prerequisites
+* KubeArchive installed and running in a cluster (xref:getting-started/installation.adoc[])
+* Know how to query the KubeArchive API (xref:getting-started/kubearchive-api.adoc[])
+
+== The KubeArchiveConfig resource
+
+To configure KubeArchive to archive or delete resources create a
+KubeArchiveConfig custom resource. KubeArchiveConfigs are limited to 1 per namespace
+and should be named `kubearchive`. KubeArchiveConfigs have this general form:
+
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+  namespace: default
+spec:
+  resources: [...]
+----
+
+`spec.resources` is a list of elements, each defining rules for a specific kind, so KubeArchive knows
+when to archive or delete them.
+
+==  `selector`: Selecting Resources
+
+Resources in the KubeArchiveConfig are defined using a `selector` which
+requires two keys: `kind` and `apiVersion`. Each entry on `spec.resources`
+requires a `selector`:
+
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+  namespace: default
+spec:
+  resources:
+    - selector:
+        apiVersion: v1
+        kind: Pod
+      ...
+    - selector:
+        apiVersion: batch/v1
+        kind: Job
+      ...
+    - selector:
+        apiVersion: apps/v1
+        kind: Deployment
+      ...
+----
+
+[TIP]
+====
+The `kind` provided in a selector must be singular. For example `kind: Pod` is valid, but `kind: Pods` is invalid.
+====
+
+With each entry on `spec.resources` KubeArchive requires one of
+`archiveWhen`, `deleteWhen` and `archiveOnDelete`. These keys accept
+a string which is an expression in the
+link:https://cel.dev[CEL language format].
+The expressions are evaluated every time a resource defined by `selector`
+changes or gets deleted. The expressions must evaluate to either true or false.
```

Comment:
I might be good to mention that `true` is a valid expression to always have something happen.